[umockdev] Unable to build

```
checking for gudev-1.0... no
configure: error: Package requirements (gudev-1.0) were not met:

No package 'gudev-1.0' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables GUDEV_CFLAGS
and GUDEV_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
umockdev failed to build: `./autogen.sh` exited with 1
```

--- a/packages/umockdev.rb
+++ b/packages/umockdev.rb
@@ -11,8 +11,12 @@ class Umockdev < Package
   depends_on 'automake' => :build
   depends_on 'libtool' => :build
   depends_on 'intltool' => :build
-  depends_on 'python27'
-
+  depends_on 'python27' => :build
+  depends_on 'glib'
+  depends_on 'libgudev'
+  depends_on 'vala'  => :build
+  
+  
   def self.build
     system "./autogen.sh"
     system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"

